MAINTAINERS.md

```diff
@@ -0,0 +1,40 @@
+## Maintainers
+Documentation for maintainers.
+
+Current maintainers: 
+- [Jake Bolam](https://github.com/jakebolam)
+- [Maximilian Berkmann](https://github.com/Berkmann18) 
+- [Tyler Benning](https://github.com/tbenning) 
+- [Jeff Wen](https://github.com/sinchang)
+
+See [the github All Contributors Core Team group](https://github.com/orgs/all-contributors/teams/core/members)
+
+## Roadmap
+- All Contributors Bot in Marketplace (Requires 250 bot installs)
+- All Contributors Website Portal?
+
+## Metrics OKR Period - Open Leaders (Mid Feb 2019 until April 2019)
+### Project & Website
+Date | Maintainers | Contributors |Stars | Website Languages  | Website Sessions | Seo Hits
+---|---|---|---|---|---|---
+**TARGET for April 201** | 4 | 40 | 2700 | 10 | 500 | 10
```

Comment:
Wanting to be able to measure our great progress!

Thinking of recording metrics, and setting targets? Thoughts?

Reply:
That's a good idea.
How often would those metrics and targets be recorded and checked?

Reply:
I'm planning to add them each Wednesday.

I wonder if we can track those automatically on a dashboard somewhere, any ideas?

Reply:
> I wonder if we can track those automatically on a dashboard somewhere, any ideas?

In regards to Lighthouse, we can use a [NodeJS script](https://github.com/GoogleChrome/lighthouse/blob/master/docs/readme.md#using-programmatically) or by using the [CLI](https://github.com/GoogleChrome/lighthouse#using-the-node-cli).

Reply:
Good idea!